wasm-engines/docker/asmble.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:19.10
```

Comment:
Could use [openjdk:8-jdk-slim-buster](https://hub.docker.com/layers/openjdk/library/openjdk/8-jdk-slim-buster/images/sha256-2d2db0073c3396b624d1783a0e23e1a317204fb729d22604a43e0fa8d1e88ae1?context=explore) and that avoids the need to install stuff.

Reply:
the binaries ship with the release, so we can just install JRE in the `bench` image IMO.